.vscode/bin/monitor-recog-fingerprints.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+if [ $# -eq 0 ]
+then
+    echo "Usage: $(basename $0) <xml fingerprint directory>"
+    exit 1
+fi
+
+if [ ! -d "$1" ]
+then
+    echo "The XML fingerprint file directory must be supplied."
+    exit 1
+fi
+
+bin/recog_verify "$1/*.xml"
+
+if ! type fswatch &>/dev/null;
+then 
+    echo "'fswatch' is required to monitor fingerprint files for changes and update the editor."
```

Comment:
This is kinda lost in the output, perhaps a leading blank line or `####################` breaks to draw attention would be helpful.